src/scss/lock.module.scss

```
@@ -0,0 +1,44 @@
+@import 'vars';
+
+.success {
+  color: $dsek-green;
+}
+
+.error {
+  color: $red;
+}
+
+.buttons {
+  display: flex;
+  justify-content: center;
+}
+
+.logoContainer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 60%;
+  height: 50%;
+  margin: 0 auto;
+}
```

Det är en bugg med styling för Configura item:et i mobilt läge. Knapparna hamnar utanför "boxen". Verkar vara att den blir tvingad till 50% i height.
Föreslår att ändra till:

```suggestion
.logoContainer {
  display: flex;
  flex-direction: column;
  justify-content: center;
  width: 60%;
  height: 50%;
  margin: 0 auto;

  @include small-mobile-only {
    height: unset;
    max-height: fit-content;
  }
}
```